### ISSUE TITLE
Provide option to run in foreground without writing to archivesspace.out

### ISSUE THREAD

**Issue by elijahgagne** [archivesspace/archivesspace#1099]
We are running ArchivesSpace in Docker. We recently found that the root filesystem ("/") in our Docker container filled up and it was because logs/archivesspace.out had used up all available space. 

As is typical with Docker, we are running the process in the foreground (so that our Docker log driver pickups up the logs). We do not want logs written inside the container, which we threat as ephemeral. 

Looking at launcher/archivesspace.sh, I see that it is doing:

```
if [ "$ARCHIVESSPACE_LOGS" = "" ]; then
    ARCHIVESSPACE_LOGS="logs/archivesspace.out"
fi

...

    "")
        # Run in foreground mode
        (cd "$ASPACE_LAUNCHER_BASE"; bash -c "$startup_cmd 2>&1 | tee '$ARCHIVESSPACE_LOGS'")
        ;;
```

I would like the option to run in foreground mode, but not write a log file in the Docker container. I plan to edit archivesspace.sh to to achieve this. Are you open to a pull request on this?

Thanks,
EWG

**Comment by elijahgagne** [comment]
Here's the naive solution I've implemented locally.

```
    foreground-nologfile)
        # Run in foreground mode without logging
        (cd "$ASPACE_LAUNCHER_BASE"; bash -c "$startup_cmd 2>&1")
        ;;
```

**Comment by mark-cooper** [comment]
@elijahgagne I'm not looking at the full context, just your snippet, but to achieve what you want have you tried setting `ARCHIVESSPACE_LOGS="/dev/null"`. If you're on Windows that won't work but maybe there's an equivalent?

**Comment by elijahgagne** [comment]
Thanks, I think that would work fine. The only downside to it is that we'd write the logs to /dev/null unnecessarily, but the overhead of that is very small so probably not a concern.

**Comment by elijahgagne** [comment]
I reviewed this with a co-work and he came up with the same solution as you before I showed him this thread. His comment was that from a performance perspective, writing to /dev/null is inconsequential, and we don't need to worry about that. But that it would be nice if the default for running Docker made it so that a log file is not written in the container. Perhaps adding ARCHIVESSPACE_LOGS="/dev/null" as an environment variable in the Dockerfile would handle that. 

Totally up to you if you want to keep this issue open. Thanks.
